shared/people/routes.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import {makeRouteDefNode, makeLeafTags} from '../route-tree'
+import People from './container'
+import profileRoutes from '../profile/routes'
+
+const peopleRoute = makeRouteDefNode({
+  ...profileRoutes.toObject(),
```

Comment:
`profileRoutes` / this might need some modifying now that the root of the stack is the people page, but I'm not totally sure. In my testing this seems to work fine as it is. If there are any places in the code that assume the root of the people tab is the current user's profile those might break.